pkgs/native_toolchain_c/lib/src/native_toolchain/msvc.dart

```diff
@@ -134,6 +152,16 @@ final Tool clIA32 = _msvcTool(
   hostArchitecture: Target.current.architecture,
 );
 
+/// The C/C++ Optimizing Compiler main executable.
+///
+/// For targeting arm64.
```

Comment:
Could be [Architecture.arm64] in the comment (ditto for the comments above).

Reply:
It would be different than other comments (for `cl`, and `clIA32`).
Do you want me to update those three comments?

Reply:
Yes please! 🙇